include/drivers/timer/nrf_rtc_timer.h

```diff
@@ -28,7 +28,7 @@ int z_nrf_rtc_timer_chan_alloc(void);
  *
  * @param chan Previously allocated channel ID.
  */
-void z_nrf_rtc_timer_chan_free(uint32_t chan);
+void z_nrf_rtc_timer_chan_free(int32_t chan);
```

Comment:
I'd suggest to keep the declarations of these functions consistent with their definitions. Although for nRF chips the `int` and `int32_t` types will be probably always equivalent, I wouldn't be surprised if some analysis tool raised at least a warning regarding this discrepancy.

Reply:
Done

Reply:
I wrote "functions" because I meant also:
- `z_nrf_rtc_timer_compare_evt_address_get`
- `z_nrf_rtc_timer_compare_int_lock`
- `z_nrf_rtc_timer_compare_int_unlock`
- `z_nrf_rtc_timer_compare_read`
- `z_nrf_rtc_timer_compare_set`

Reply:
sorry, missed that. should look better now.